src/pages/dashboard/Dashboard.js

```diff
@@ -1,9 +1,17 @@
 import "./Dasboard.css";
-import {withAuthenticationRequired} from "@auth0/auth0-react";
+import {useAuth0, withAuthenticationRequired} from "@auth0/auth0-react";
 import PermanentDrawerLeft from "../../component/drawer/PermanentDrawerLeft";
+import BackHandOutlinedIcon from "@mui/icons-material/BackHandOutlined";
+import DirectionsOutlinedIcon from "@mui/icons-material/DirectionsOutlined";
 
 function Dashboard() {
-  return <PermanentDrawerLeft />;
+  const {logout} = useAuth0();
```

Comment:
We can move `navItems` outside of component so to avoid reinitialisation when component re-renders. 

Declare and define it before above dashboard